src/Button.js

```diff
@@ -13,12 +13,15 @@ class Button extends Component {
   }
 
   componentDidMount() {
-    const { tooltip, tooltipOptions } = this.props;
-    if (
-      typeof $ !== 'undefined' &&
-      (typeof tooltip !== 'undefined' || typeof tooltipOptions !== 'undefined')
-    ) {
-      $(this._btnEl).tooltip(tooltipOptions);
+    if (!M) return;
+
+    const { tooltip, tooltipOptions, fab } = this.props;
+    if (tooltip || tooltipOptions) {
```

Comment:
If `tooltipOptions` is defined but there is no actual `tooltip` text should it still appear?

Reply:
You're right :)

Reply:
Fixed :)